components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAO.java

```diff
@@ -47,6 +47,8 @@ default AccessTokenDO getRefreshToken(String refreshToken) throws IdentityOAuth2
     void revokeOAuthConsentByApplicationAndUser(String username, String tenantDomain, String applicationName)
             throws IdentityOAuth2Exception;
 
+    void revokeOAuthConsentsByApplication(String applicationName, String[] tenantDomain) throws IdentityOAuth2Exception;
```

Comment:
make this a default method so that the existing implementations will not break

Reply:
Addressed in [6918b82](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/6918b828f239ba766e20f66e6d0af997d891413a)